common/src/test/java/io/pravega/common/util/ConfigurationOptionsExtractorTests.java

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright (c) Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.common.util;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertSame;
+
+public class ConfigurationOptionsExtractorTests {
+
+    @Test
+    public void testExtractInteger() {
+        String propertyName = this.getClass().getSimpleName() + ".testExtractInteger." + ".test.property";
+        System.setProperty(propertyName, "5");
+        assertSame(5, ConfigurationOptionsExtractor.extractInt(propertyName,
+                "test_property_1", 1));
+        System.clearProperty(propertyName);
```

Comment:
Put this in a `finally` block to ensure it is always cleaned, regardless of the outcome of your test.

Reply:
Sure, that's a good idea too. I've pushed this change.